cita-network/src/node_manager.rs

```diff
@@ -913,12 +1060,41 @@ impl BroadcastReq {
             self.msg,
             self.key
         );
-        let msg_bytes: Vec<u8> = self.msg.try_into().unwrap();
 
-        let mut buf = Vec::with_capacity(CITA_FRAME_HEADER_LEN + self.key.len() + msg_bytes.len());
-        pubsub_message_to_network_message(&mut buf, Some((self.key, msg_bytes)));
-        if let Some(ref mut ctrl) = service.service_ctrl {
-            let _ = ctrl.filter_broadcast(TargetSession::All, TRANSFER_PROTOCOL_ID, buf.into());
+        let mut info = NetMessageUnit::default();
+        info.key = self.key;
+        info.data = self.msg.try_into().unwrap();
+        info.addr = service.peer_key;
+        info.version = service.self_version;
+        service.self_version += 1;
+
+        let cur_status: String = routing_key!(Synchronizer >> Status).into();
+        let auth_txs: String = routing_key!(Auth >> Request).into();
+        let mut dst = TargetSession::All;
+        /* Synchronizer >> Status for declare myself status,only to neighbors
```

Comment:
Check this:
https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md#comments

> Prefer line comments (//) to block comments (/* ... */).

> When using line comments there should be a single space after the opening sigil.

Reply:
done